tensorflow/python/kernel_tests/check_ops_test.py

```diff
@@ -1592,6 +1593,11 @@ def test_rank_zero_rank_one_size_one_equivalence(self):
         (rank_zero, (1,)),
     ])
 
+  @test_util.run_in_graph_and_eager_modes
+  def test_sparse_tensor_input(self):
+    A = array_ops.ones([2, 2], name="rank_two")
```

Comment:
I believe we want the tensor name to be lower snake case to abide by python code style guidelines.

Reply:
changed as request :)